versioned_docs/version-v1.3/host/host.md

```diff
@@ -21,10 +21,36 @@ Because Harvester is built on top of Kubernetes and uses etcd as its database, t
 
 ## Node Maintenance
 
+In the following scenarios, you plan to migrate/shutdown the workloads from one node and also possibly shutdown the node.
+
+- Replace/add/remove hardware
+
+- Change the network setting
+
+- Troubleshooting
+
+- Remove a node
+
+Harvester provides `Node Maintenance` feature to run a series of checks and operations automatically.
+
 For admin users, you can click **Enable Maintenance Mode** to evict all VMs from a node automatically. It will leverage the `VM live migration` feature to migrate all VMs to other nodes automatically. Note that at least two active nodes are required to use this feature.
 
 ![node-maintenance.png](/img/v1.2/host/node-maintenance.png)
 
+After a while the target node will enter maintenance mode successfully.
+
+![node-enter-maintenance-mode.png](/img/v1.3/troubleshooting/node-enter-maintenance-mode.png)
+
+:::info important
+
+Check those [known limitations and workarounds](../troubleshooting/host.md#an-enable-maintenance-mode-node-stucks-on-cordoned-state) before you click this menu or when you have encountered some issues.
+
+If you have attached any volume to this node manually, it may block the node maintenance, check the section [Manually Attached Volumes](../troubleshooting/host.md#manually-attached-volumes) and set a proper global option.
```

Comment:
```suggestion
If you have attached any volume to this node manually, it may block the `Node Maintenance`, check the section [Manually Attached Volumes](../troubleshooting/host.md#manually-attached-volumes) and set a proper global option.
```